sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -424,11 +419,15 @@ export interface SubscriptionResponse extends SubscriptionProperties, Response {
 // @public
 export interface SubscriptionRuntimeProperties {
     accessedAt: Date;
+    activeMessageCount: number;
     createdAt: Date;
-    messageCount: number;
-    messageCountDetails?: MessageCountDetails;
+    deadLetterMessageCount: number;
+    scheduledMessageCount: number;
     subscriptionName: string;
     topicName: string;
```

Comment:
Looks like the only difference between SubscriptionRuntimeProperties and QueueRuntimeProperties is the names.
```ts
// SubscriptionRuntimeProperties
{
   subscriptionName: string;
   topicName: string;
}
// QueueRuntimeProperties
{
    name:string;
}
```

Should we consider refactoring/moving repeated ones to a common interface?

Reply:
I personally wouldnt do so. For me, this falls under the category of over-optimization with not much gains.

Reply:
Fine!